README.md

```diff
@@ -1,46 +1,65 @@
-# ros2_shm_vision_demo
+# shm_vision_demo
 
 Demonstrate how to use shared memory with image processing algorithms.
 
 Note this is highly experimental and requires much clean up. It may still serve as an inspiration how to set up a computer vision app in ROS 2 with shared memory for high transmission efficiency.
 
 ## Requirements
 
-Only works with ROS 2 Rolling.
+As of today this package only supports ROS 2 Rolling.
+
 ROS 2 Galactic is not supported due to some cmake command changes.
 
-Also require yolo3 object detection parameters found here.
+Also require yolov3 object detection parameters which are automatically downloaded at build time using the `scripts/download_yolo_weights.sh` script.
 
 ## Installation
 
 TODO: more detail
 
 1. Install ROS 2 Rolling
 
-2. Clone the ros2_shm_vision_demo repo into the src folder of the ROS 2 Rolling installation.
+2. Clone the `shm_vision_demo` repo into your `colcon_ws/src` directory
+
+3. Clone `iceoryx` repo into your `colcon_ws/src` directory - parallel with the repo you just cloned in step number two
 
-3. Build using `colcon build --packages-up-to ros2_shm_vision_demo`
+4. Build using `colcon build --packages-up-to shm_vision_demo`.  While building it will automatically fetch the `yolov3.weights` file from https://pjreddie.com/darknet/yolo/ and copy them to the `shm_vision_demo/config` directory. The weights file is fairly large so it is not included in the repository but automatically downloaded during the build.
 
-4. Create a folder vision_config in your ROS 2 Rolling workspace (parallel to src). Copy the cyclonedds.xml and roudi_config.toml to this folder. (This should be automatically installed properly later but currently it is not).
+The following files are needed in the `config/` directory:
+  - `coco.names`
+  - `yolov3.cfg`
+  - `yolov3.weights` (YOLOv3-320 for the right image size)
 
-5. Obtain the yolo3 config files from https://pjreddie.com/darknet/yolo/ and copy them to the same folder (The application currently uses hardcoded paths, to be changed later). The following files are needed coco.names (config folder), yolov3.cfg and yolov3.weights (YOLOv3-320 for the right image size). The weights file is fairly large so it is not included in the repository.
 
-6. Adapt the `ros2_ws` and `video` variables as in the of the tmux scripts (scripts folder) as needed (TODO install scripts, relative paths). `ros2_ws` needs to point to the ROS 2 rolling workspace and `video` to some full hd video.
+5. Adapt the `ros2_ws` and `video` variables in the `scripts/demo.sh` and `scripts/shm_demo.sh` scripts, located in the `scripts` folder as needed. `ros2_ws` needs to point to the ROS 2 rolling workspace and `video` to some full hd video.
```

Comment:
Adapting video is now optional with the changed script as far as I can see. It will use the argument as video name or the camera otherwise? It is of course still possible to change the default video to some video as well. But maybe this is implicit and in any case I think it is understandable from the script and the readme.

Reply:
You are correct! i'll fix this in the README